### ISSUE TITLE
Language about grey areas

### ISSUE THREAD

**Issue by alanwsmith** [hachyderm/community#190]
The usage docs have two similar statements:

---

If your intended post is in a grey area for one of these rules, then err on the side of caution and reach out to the mods via our [GitHub issue tracker](https://github.com/hachyderm/community/issues).

---

Hachydermians may need to discuss some topics on the above "No" list from time to time. Whether it is sharing current news, feelings about current news, personal experiences and/or impacts, or making historical connections the fact is that the world is a bit on fire. Conversations from these perspectives are welcome, however: use content warnings on text and media whenver discussing or sharing anything that might fall under these rules. If you are unsure, use a content warning as that is why the feature is there. 

---

The first one is explicit about talking to a mod. I interpret the second one to say it's okay to post things that would otherwise break the rules as long as they are behind CWs. 

Feels like that could lead to some uncertainty. 

I drafted this as a first draft of a potential replacement if y'all are interested:

---

Like every rule set, the edges can be hard to determine. 
Hachydermians may occasionally wish to discuss news, 
personal experiences, historical connections, or similar 
topics that would otherwise conflict with the rules. 
**Conversations in good faith are welcome, but must always 
be posted behind content warnings on both text and media.** 

However, those exceptions are not limitless. 

> If your intended post is in a grey area and you have 
> any doubt about it's acceptability, then err on the side 
> of caution and reach out to the mods via our [GitHub issue tracker](https://github.com/hachyderm/community/issues)

**Comment by alanwsmith** [comment]
Added some more text I missed the first time:

---

Like every rule set, the edges can be hard to determine. 
Hachydermians may occasionally wish to discuss news, 
personal experiences, historical connections, or similar 
topics that would otherwise conflict with the rules. 
**Conversations in good faith are welcome, but must always 
be posted behind Content Warnings (CWs) on both text and media.** 

Content Warnings are how we, as a community, can give space
to as many people as possible. We can serve those who need
to discuss sensitive topics while protecting those who are
actively experiencing, or recovering from, the topic in the 
discussion. 

To put it another way: **One person's case study is another 
person's life and trauma.** Treat these conversations with respect.

Keep in mind though, the exceptions are not limitless. 

> If your intended post is in a grey area and you have 
> any doubt about it's acceptability, then err on the side 
> of caution and reach out to the mods via our [GitHub issue tracker](https://github.com/hachyderm/community/issues)

**Comment by quintessence** [comment]
Ah thank you for your attention to the docs! Would you mind sending a DM my way (on Hachyderm is fine) so I can make sure you get what you need to fully contribute (if you're interested) :)

**Comment by quintessence** [comment]
👋 Since we synced up I'll go ahead and close this issue. We'll keep this internally for part of our docs. Thank you again!
